.bandit.yaml

```
@@ -0,0 +1,14 @@
+# Keep Bandit's assert rule in production code, but allow pytest-style asserts in tests.
+assert_used:
+  skips:
+    - "*_test.py"
+    - "*test_*.py"
+    - "test_*.py"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)